src/scrabble.js

```diff
@@ -1,6 +1,49 @@
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.word = word
+    this.onePoint = ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't']
+    this.twoPoints = ['d', 'g']
+    this.threePoints = ['b', 'c', 'm', 'p']
+    this.fourPoints = ['f', 'h', 'v', 'w', 'y']
+    this.fivePoints = ['k']
+    this.eightPoints = ['j', 'x']
+    this.tenPoints = ['q', 'z']
+  }
 
+  score () {
+    let finalScore = 0
+    let subScore = 0
+    let multiply = 1
+    // check if the argument is a string
+    if (typeof this.word !== 'string') {
+      return finalScore
+    } else {
+      const wordArr = this.word.toLowerCase().split('')
+      for (let i = 0; i < wordArr.length; i++) {
+        // check if there is curly bracket or square bracket
+        // If so, change the multiply to 2 or 3. If not, multiply remains 1
+        if (wordArr[i] === '{') {
+          multiply = 2
+        } else if (wordArr[i] === '}') {
+          multiply = 1
+        } else if (wordArr[i] === '[') {
+          multiply = 3
+        } else if (wordArr[i] === ']') {
+          multiply = 1
```

Comment:
what would happen if there is a double letter within a double scored word? The multiple would return to one after the double letter

if (wordArr[i] === '{') {
          multiply = multiply * 2
        } else if (wordArr[i] === '}') {
          multiply = multiply / 2
        } else if (wordArr[i] === '[') {
          multiply = multiply * 3
        } else if (wordArr[i] === ']') {
          multiply = multiple / 3